### PR TITLE
added the webcam feature in detector.py

### PR DESCRIPTION
**Pull request by parikshit14**
I have added to directly enable the webcam during the inference, did some changes in detector.py and yolo.py files. As of now the code is #commented# if it is satisfactory enough Kindly merge the PR.

### CODE REVIEW

**Comment by AntonMu** [comment]
Hi @parikshit14,

Thank you for your contribution. I appreciate it. You are the first external contributor and I wrote down some guidelines for contributing to the repo. 

Check them out here: [CONTRIBUTING.md](https://github.com/AntonMu/TrainYourOwnYOLO/blob/master/CONTRIBUTING.md)!

File: 3_Inference/Detector.py [hunk 1]
```diff
@@ -293,5 +294,13 @@ def get_parent_dir(n=1):
...
             )
         )
+    # for Webcam uncomment the below code and the one in 2_Training/src/keras_yolo3/yolo.py
+    """
+    start = timer()
+    detect_video(yolo)
+
+    end = timer()
+    print("Processed {} videos in {:.1f}sec".format( len(input_video_paths), end - start))
+    """
```
Comment (AntonMu) [inline 3_Inference/Detector.py:304]: Instead of uncommenting, make it an (optional) CLI argument

File: 2_Training/src/keras_yolo3/yolo.py [hunk 2]
```diff
@@ -286,3 +286,68 @@ def detect_video(yolo, video_path, output_path=""):
...
+    #)
+    #isOutput = True if output_path != "" else False
+    #if isOutput:
+    #    print(
+    #        "Processing {} with frame size {} at {:.1f} FPS".format(
+    #            os.path.basename(video_path), video_size, video_fps
+    #        )
+    #    )
+    #    print("!!! TYPE:", type(output_path), type(video_FourCC), type(video_fps), type(video_size))
+    #    out = cv2.VideoWriter(output_path, video_FourCC, video_fps, video_size)
```
Comment (AntonMu) [inline 2_Training/src/keras_yolo3/yolo.py:312]: Remove commented code

File: 2_Training/src/keras_yolo3/yolo.py [hunk 3]
```diff
@@ -286,3 +286,68 @@ def detect_video(yolo, video_path, output_path=""):
...
+            org=(3, 15),
+            fontFace=cv2.FONT_HERSHEY_SIMPLEX,
+            fontScale=0.50,
+            color=(255, 0, 0),
+            thickness=2,
+        )
+        cv2.namedWindow("result", cv2.WINDOW_NORMAL)
+        cv2.imshow("result", result)
+        #if isOutput:
+        #    out.write(result[:, :, ::-1])
```
Comment (AntonMu) [inline 2_Training/src/keras_yolo3/yolo.py:347]: same as above

**Review by AntonMu — CHANGES_REQUESTED** [review]

File: 3_Inference/Detector.py (hunk 1 above)
Comment (parikshit14) [reply 3_Inference/Detector.py:304]: Ok i' ll to do all these changes ASAP

**Pull request closed without merging**